src/pkg/selectors/scopes.go

```diff
@@ -351,7 +351,11 @@ func reduce[T scopeT, C categoryT](
 			continue
 		}
 
-		pv := dc.pathValues(repoPath, *ent)
+		pv, err := dc.pathValues(repoPath, *ent)
+		if err != nil {
+			el.AddRecoverable(clues.Wrap(err, "getting path values").WithClues(ctx))
```

Comment:
it occurs to me that we don't have any specific item identification for errors added in this loop.  I can't comment up on that line, but could you add something like this on line 311?
```go
ictx := clues.Add(ctx, "short_ref", ent.ShortRef)
```
and then propagate that inner context within the other calls.
```suggestion
			el.AddRecoverable(clues.Wrap(err, "getting path values").WithClues(ictx))
```